README.md

```diff
@@ -1,2 +1,260 @@
+There are some great resources to learn about these concepts, and they are all mentioned in the documentation of each individual shader. For all of the three-dimensional fractals, the resources I used were:
+- [Inigo Quilez](http://iquilezles.org)​
+- [Sebastian Lague](https://www.youtube.com/user/Cercopithecan)
+- [Code Parade](https://www.youtube.com/channel/UCrv269YwJzuZL3dH5PCgxUw)
+
+### Mandelbulb
+
+![gif](./media/mandelbulb.gif)
+
+📌 This fractal was first constructed by Daniel White and Paul Nylander in 2009, using spherical coordinates. It is not, however, a canonical three-dimensional Mandelbrot set. Such structure does not exist, since there is no three-dimensional analogue to the two-dimensional complex set of numbers. It is possible to construct a canonical Mandelbrot Set in four dimensions using quaternions and bicomplex numbers, but the Mandelbulb is different to that.
+
+➕ It has fractal (Hausdorff) dimension of:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;D=3"/>
+
+This is, however, only been conjectured, and has not been proven as of the creation of this repository.
+
+🎨 The Mandelbulb is defined as the set o points c in  ℝ<sup>3</sup> that, under the iteratoin of the function:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;f_{c}(v)=v^n+c"/>
+
+with v starting at (0, 0, 0) does not diverge.\
+The n<sup>th</sup> power of the vector v = (x, y, z) in ℝ<sup>3</sup> is defined as:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;v^n:=r^n(\sin{(n\theta)}\cos{(n\phi)},\sin{(n\theta)}\sin{(n\phi)},\cos{(n\theta)})"/>
+
+Where:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;r=\sqrt{x^2+y^2+z^2}"/>\
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;\theta=\arccos{\frac{z}{y}}"/>\
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;\phi=\arctan{\frac{z}{r}}"/>
+
+The Mandelbulb shown on the video is one constructed with n = 8. Different values of n result in different structures. The flow effect on the video is a result of variying the spherical coordinates of each point continuously.\
+Probably the most common way of constructing this fractal is by using thechniques such as ray-tracing or ray-marching (as used on the shader for this fractal).
+
+### Mandelbox
+
+![gif](./media/mandelbox.gif)
+
+📌 A Mandelbox is not a specific fractal, unlike most of the ones covered so far. It can take many forms, but, generaly, can be described as fractal with a boxlike shape, and it was first found by Tom Lowe in 2010.\
+It is similar to the Mandelbrot set in the same way the Mandelbulb is similar to it: it is defined as a set of values of a parameter such that the origin (in ℝ<sup>3</sup>, (0, 0, 0)) does not diverge under inifinite iteration of a certain geometrical transformation. It can can also be defined as a map of continuous Julia sets, but can be generalized to any dimension, unlike the Mandelbrot set.
```

Comment:
```suggestion
📌 A Mandelbox is not a specific fractal, unlike most of the ones covered so far. It can take many forms, but, generally, can be described as fractal with a boxlike shape, and it was first found by Tom Lowe in 2010.\
It is similar to the Mandelbrot set in the same way the Mandelbulb is similar to it: it is defined as a set of values of a parameter such that the origin (in ℝ<sup>3</sup>, (0, 0, 0)) does not diverge under infinite iteration of a certain geometrical transformation. It can can also be defined as a map of continuous Julia sets, but can be generalized to any dimension, unlike the Mandelbrot set.
```